apps/prairielearn/assets/scripts/instructorRequestCourseLti13.ts

```diff
@@ -0,0 +1,35 @@
+import { decodeData, onDocumentReady } from '@prairielearn/browser-utils';
+
+onDocumentReady(() => {
+  const courseRequestLti13Info = decodeData('courseRequestLti13Info');
+
+  if (courseRequestLti13Info === null) {
+    return;
+  }
+
+  const Lti13Modal = new bootstrap.Modal(document.getElementById('lti13FillModal'), {});
+
+  Lti13Modal.show();
+
+  const autoFillLti13Button = document.getElementById('fillCourseRequestLti13');
+
+  autoFillLti13Button?.addEventListener('click', () => {
+    const courseRequestForm = document.querySelector(
+      'form[name="course-request"]',
+    ) as HTMLFormElement;
```

Comment:
You lose track of the nullability by casting the return value.

```suggestion
    const courseRequestForm = document.querySelector<HTMLFormElement>(
      'form[name="course-request"]',
    );
```